models/articles.model.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection.js");
+
+exports.selectArticleById = (article_id) => {
+    if (!/-?\d+$/.test(article_id)) {
```

Comment:
Clever.  But not necessary if you have psql Error handlers!

Reply:
I changed it to handle the error code in the error handler, i hope its right 😟 thanks for the feedback